test/Microsoft.Azure.Devices.Edge.Test/GenericMqtt.cs

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Test
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.Test.Common;
+    using Microsoft.Azure.Devices.Edge.Test.Common.Config;
+    using Microsoft.Azure.Devices.Edge.Test.Helpers;
+    using Microsoft.Azure.Devices.Edge.Util.Test.Common.NUnit;
+    using NUnit.Framework;
+
+    [EndToEnd]
```

Comment:
Adding a comment block here with some of the text from the description section of this PR might be useful here to document what this class does.